src/App.css

```diff
@@ -0,0 +1,6 @@
+button {
+  margin-left: 5px;
+}
+li {
+  padding-bottom: 5px;
+}
```

Comment:
Good initiative in starting to style your app! Looking ahead, styles are covered in lesson 3.1, fyi